### PR TITLE
Bug 1776855: Fixes negative values in capacity breakdown

### PR DESCRIPTION
**Pull request by afreen23**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1776855

Before:
Screenshot from 2019-12-27 16-35-46

After:
Screenshot from 2019-12-27 16-37-01

Co-authored-by: Kanika <kmuraraka@redhat.com>
Signed-off-by: Afreen Rahman <afrahman@redhat.com>

### CODE REVIEW

**Comment by afreen23** [comment]
e2e-gcp-console is broken now.
Wait for fix to add lgtm

File: frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/breakdown-card/utils.tsx [hunk 1]
```diff
@@ -19,6 +19,7 @@ const addOthers = (
     name: 'Other',
     color: Colors.OTHER,
     label: humanize(others).string,
+    fill: 'rgb(96, 98, 103)',
```
Comment (a2batic) [inline frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/breakdown-card/utils.tsx]: It's good to add it to constants and used it from there.

File: frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/breakdown-card/utils.tsx [hunk 2]
```diff
@@ -48,6 +49,7 @@ export const addAvailable = (
       link: '',
       color: '',
       label: humanize(availableInBytes).string,
+      fill: '#b8bbbe',
```
Comment (a2batic) [inline frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/breakdown-card/utils.tsx]: (Same) It's good to add it to constants and used it from there.

**Comment by afreen23** [comment]
/hold 
as e2e tests are broken

**Comment by spadgett** [comment]
/hold cancel
/retest
#3847 has fixed the e2e tests

**Pull request merged**